backend/inspirehep/editor/views.py

```diff
@@ -379,3 +368,58 @@ def remove_editor_lock(endpoint, pid_value):
     )
     editor_soft_lock.remove_lock()
     return jsonify(success=True)
+
+
+@blueprint.route("/authorlist/url", methods=["POST"])
+@login_required
```

Comment:
this login will allow any kind of user, we should use `@login_required_with_roles([Roles.cataloger.value])`